recipes/ucc/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+./autogen.sh
+./configure --prefix=$PREFIX --with-ucx=$PREFIX --with-cuda=$PREFIX
```

Comment:
Personally prefer to quote paths just in case there is blank space or weird characters. Should be pretty unlikely (especially on CI in conda-forge). Still seems like a safe thing to do

Am curious whether the build system can just detect the others without needing to specify them. Do you know? If so, we should simplify like so

```suggestion
./configure --prefix="${PREFIX}"
```

If we need to specify them in all cases, maybe we can just quote them all like so

```suggestion
./configure --prefix="${PREFIX}" --with-ucx="${PREFIX}" --with-cuda="${PREFIX}"
```

Reply:
Suggested change #1 does not work: "configure: error: UCX support is requested but UCX packages cannot be found"